app/i18n.js

```diff
@@ -7,16 +8,19 @@
  *   script `extract-intl`, and must use CommonJS module syntax
  *   You CANNOT use import/export in this file.
  */
-const addLocaleData = require('react-intl').addLocaleData; //eslint-disable-line
-const enLocaleData = require('react-intl/locale-data/en');
-const deLocaleData = require('react-intl/locale-data/de');
+if (!Intl.PluralRules) {
+  require('@formatjs/intl-pluralrules/polyfill');
+  require('@formatjs/intl-pluralrules/dist/locale-data/de'); // Add locale data for de
+}
+
+if (!Intl.RelativeTimeFormat) {
+  require('@formatjs/intl-relativetimeformat/polyfill');
+  require('@formatjs/intl-relativetimeformat/dist/locale-data/de'); // Add locale data for de
+}
```

Comment:
The sample app has both de and en so I believe you're missing a couple lines above:

```js
if (!Intl.PluralRules) {
  require('@formatjs/intl-pluralrules/polyfill');
  require('@formatjs/intl-pluralrules/dist/locale-data/de'); // Add locale data for de
  require('@formatjs/intl-pluralrules/dist/locale-data/en'); // Add locale data for en
}

if (!Intl.RelativeTimeFormat) {
  require('@formatjs/intl-relativetimeformat/polyfill');
  require('@formatjs/intl-relativetimeformat/dist/locale-data/de'); // Add locale data for de
  require('@formatjs/intl-relativetimeformat/dist/locale-data/en'); // Add locale data for en
}
```

No?

Reply:
You can read [here](https://www.npmjs.com/package/@formatjs/intl-relativetimeformat#usage) that the en locale is the default locale, the example shows the de locale being imported explicitely.

Reply:
I don't see any mention of `en` being the default locale at this link. IMO, if `@formatjs/intl-relativetimeformat/dist/locale-data/en` didn't need to be included, it wouldn't exist in the `locale-data` folder but it does.

Reply:
I guess I was not imagining it, they removed it from the README.md a while ago: https://github.com/formatjs/formatjs/commit/cda3cf2090abf6a46e293ad049eb7f588f9a66eb#diff-70b3d6472c5178da3042f45cb893ef56

I am updating react-intl and relevant dependencies again, since there were a lot of new version releases in the mean time, I will keep you updated.

Reply:
Thanks @mhatvan! Sorry again for the delay on my part!